package.json

```diff
@@ -5,33 +5,40 @@
     "node": ">=12.9.0"
   },
   "workspaces": [
+    "packages/params",
+    "packages/utils",
+    "packages/types",
+    "packages/spec-test-util",
+    "packages/config",
+    "packages/api",
+    "packages/db",
+    "packages/state-transition",
+    "packages/light-client",
+    "packages/validator",
+    "packages/flare",
+    "packages/fork-choice",
+    "packages/beacon-node",
```

Comment:
This hardcoded list is sorted topologically. Yarn v1 does not support topological inference out of the box, so this does the trick. Long term we can switch to newer versions of yarn that do this automatically.